### DISCUSSION TITLE
v2.0 release on pip?

### DISCUSSION THREAD

**Discussion by NabeelUppel** [Rapptz/discord.py#7478] [Q&A] [answered]
I was just wondering when do you plan to release version 2 on pip? I want to use travis ci and another library (distest) but I would also like to use threads which is only available on v2.0
Thanks in advance.

**Comment by imvaskel** [comment, marked as answer]
There is currently no planned release date for 2.0, but you can download it from master with ``git+https://github.com/rapptz/discord.py`` if you would like to use the development version. Do be warned that this can have breaking changes at any time.

**Reply by NabeelUppel** [reply]
Is there any other way to use threads api that does not involve the dev branch?

**Reply by imvaskel** [reply]
no. discord is only sending thread dispatches on API V9.

**Reply by NabeelUppel** [reply]
Thanks for the replies

**Reply by TechStudent10** [reply]
The project has been revived, so a v2 release might be on the way.
